src/metabase/api/pulse.clj

```diff
@@ -122,20 +137,24 @@
                     :dashboard_id        dashboard_id
                     :parameters          parameters}]
     (db/transaction
-      ;; Adding a new pulse at `collection_position` could cause other pulses in this collection to change position,
```

Comment:
old indentation was correct https://guide.clojure.style/#body-indentation

Reply:
old indentation was 1 space, isn't that wrong? and the `:style/indent` metadata on `db/transaction` is 0